pkg/cmd/list/edge_applications/edge_applications.go

```diff
@@ -0,0 +1,116 @@
+package edge_applications
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/fatih/color"
+
+	"github.com/MakeNowJust/heredoc"
+	table "github.com/MaxwelMazur/tablecli"
+	"github.com/aziontech/azion-cli/messages/general"
+	msg "github.com/aziontech/azion-cli/messages/list/edge_applications"
+	api "github.com/aziontech/azion-cli/pkg/api/edge_applications"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	opts := &contracts.ListOptions{}
+
+	cmd := &cobra.Command{
+		Use:           msg.ListUsage,
+		Short:         msg.ListShortDescription,
+		Long:          msg.ListLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true, Example: heredoc.Doc(`
+            $ azion list edge_applications
```

Comment:
fix the examples to show new command formats
azion list edge-application
etc...